### DISCUSSION TITLE
Crowdin question

### DISCUSSION THREAD

**Discussion by xerta555** [LSPosed/LSPosed#1113] [Q&A] [answered]
Hi :)

I see that new EN strings doesn't exist on Crowdin for LSPosed project, also i see that the XML file for EN strings contain 195 lines, and for exemple the FR XML file contain 179.

Is it normal that new strings are not automatically added to Crowdin?

Kind regards.

**Comment by yujincheng08** [comment, marked as answer]
https://github.com/LSPosed/LSPosed/blob/master/.github/workflows/crowdin.yml

**Reply by xerta555** [reply]
What?

**Reply by yujincheng08** [reply]
> What?

it's a github action that automatically sync strings to crowdin.

**Reply by xerta555** [reply]
Ok, and what's the sync frequency?

**Reply by yujincheng08** [reply]
> Ok, and what's the sync frequency?

whenever the string changed.

**Reply by xerta555** [reply]
> whenever the string changed.

I ask because new strings are not sync on Crowdin, i can see them in the EN xml strings file.

**Reply by yujincheng08** [reply]
name it. we will check

**Reply by xerta555** [reply]
> 
> 
> name it. we will check

Line 169: settings_translation

I don't see language part (the language part in the settings of the app), so 2 strings lines, and also for "Dark theme".

I think that's it for now, as i'm having no trouble at the moment getting the LSposed subsystem to work on my device, i don't have the error messages so i don't have an immediate way of knowing if other translations are also impacted.

**Reply by yujincheng08** [reply]
settings_translation is NOT missing: https://lsposed.crowdin.com/translate/63c7877a25dd553affc478efd218bdab/31/en-fr/33?filter=basic&value=3#q=settings_translation

And "Dark Theme" is from upstream RikkaX so it cannot be translated by us.

**Reply by xerta555** [reply]
> 
> 
> settings_translation is NOT missing: [lsposed.crowdin.com/translate/63c7877a25dd553affc478efd218bdab/31/en-fr/33?filter=basic&value=3#q=settings_translation](https://lsposed.crowdin.com/translate/63c7877a25dd553affc478efd218bdab/31/en-fr/33?filter=basic&value=3#q=settings_translation)
> 

Ok i have done the translation.

> And "Dark Theme" is from upstream RikkaX so it cannot be translated by us.

Form here: https://github.com/RikkaApps/RikkaX if i understand correctly ?
I asked that because "Dark theme" isn't translated, and "Système de suivi" translation is very....poor. Final users will understand approximativly, they will guess.

**Reply by yujincheng08** [reply]
> Form here: https://github.com/RikkaApps/RikkaX if i understand correctly ?
yup. Translate there.

**Reply by xerta555** [reply]
> 
> 
> > Form here: [RikkaApps/RikkaX](https://github.com/RikkaApps/RikkaX) if i understand correctly ?
> > yup. Translate there.

Ok translations fixs sended as pr on this repo so just have to.

How are you doing with the development of LSPosed? I keep seeing your pr 🙃

**Reply by xerta555** [reply]
@yujincheng08 Hi, my PR on the Rikka repo https://github.com/RikkaApps/RikkaX/commit/798884d501bbcd1cf483c6e8baa3586f2de86ed7 has been merged 4 days ago, i just update framework and app on my device but even after a reboot i see that the translations changes haven't been updated.

Could you take a look in that please?
Thanks.

**Reply by yujincheng08** [reply]
You should wait for @RikkaW to make a new rikkax release.
